### PR TITLE
Add description how to use a Node UI discovery

### PR DESCRIPTION
**Pull request by soffokl**
This PR requires https://github.com/mysteriumnetwork/node/pull/1135 to be merged.

### CODE REVIEW

File: docs/user-guide/postinstall.md [hunk 1]
```diff
@@ -124,3 +124,80 @@ To get a detailed log for Mysterium node service, you can use the following comm
 ```
 sudo journalctl -u mysterium-node.service
 ```
+
+## Built-in UI and Node LAN discovery
+
+It is possible to start a Mysterium node with a built-in UI server to do the basic configuration.
+
+The `--ui.enable` flag should be used to start a node with built-in UI.
```
Comment (tadaskay) [inline docs/user-guide/postinstall.md]: Do we really need this switch? Why can't we simply start with UI?

File: docs/user-guide/postinstall.md [hunk 2]
```diff
@@ -124,3 +124,80 @@ To get a detailed log for Mysterium node service, you can use the following comm
...
+
+Mysterium Node with enabled UI will announce Bonjour service to all hosts within the LAN.
+
+To access a Mysterium Node UI you can open in browser link similar to following:
+
+[http://<hostname>.:4449](http://<hostname>.:4449)
+
+`hostname` here is a hostname of the Mysterium Node. For example, if you are running a Raspberry Pi Mysterium Node with a default hostname you can use the following link:
+
+[http://raspberrypi.:4449](http://raspberrypi.:4449)
```
Comment (tadaskay) [inline docs/user-guide/postinstall.md]: So basically there's no way to advertise it so that it is displayed automatically as in Windows, right?

**Review by tadaskay — APPROVED** [review]

**Review by vkuznecovas — DISMISSED** [review]

File: docs/user-guide/postinstall.md (hunk 1 above)
Comment (vkuznecovas) [reply docs/user-guide/postinstall.md]: To get rid of it we need to do conditional compiling to exclude the whole ui from mobile devices.

Comment (tadaskay) [inline docs/user-guide/postinstall.md]: Changed this to be true by default in https://github.com/mysteriumnetwork/node/pull/1141
Either remove the section, or inverse it (provide a way to disable it via `--ui.enable=false`)

**Review by tadaskay — CHANGES_REQUESTED** [review]

File: docs/user-guide/postinstall.md (hunk 2 above)
Comment (soffokl) [reply docs/user-guide/postinstall.md]: Yeah, this nice showing new devices in Explorer with SSDP works only for Windows.

File: docs/user-guide/postinstall.md (hunk 1 above)
Comment (soffokl) [reply docs/user-guide/postinstall.md]: Removed it.

**Review by iberflow — DISMISSED** [review]

**Review by zolia — DISMISSED** [review]

**Review by tadaskay — APPROVED** [review]

**Pull request merged**